.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,61 @@
+name: "Create Release PR"
+
+on:
+  workflow_dispatch:
+    inputs:
+      versionNumber:
+        description: 'Version Number'
+        required: true
+
+env:
+  BRANCH_NAME: "release/${{ github.event.inputs.versionNumber }}"
+  VERSION_NUMBER: ${{ github.event.inputs.versionNumber }}
+
+jobs:
+  create-release-pr:
+    runs-on: macos-latest
+    timeout-minutes: 5
+    name: "Create Release PR"
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v4
+      with:
+        ref: main
+    - name: Setup SSH Keys
+      uses: webfactory/ssh-agent@v0.7.0
+      with:
+        ssh-private-key: |
+          ${{ secrets.APOLLO_IOS_DEV_DEPLOY_KEY }}
+    - name: Configure Git
+      uses: ./.github/actions/configure-git
+    - name: Run Commands
```

Comment:
Can we make the name here describe what commands are being run. Or break this into different steps for each command?

Reply:
Sure I can break them out to separate commands